packages/avatars/demo/~patterns/stories/StatusMenuStory.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Story } from '@storybook/react';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+import { Dropdown, Trigger, Menu, Item } from '@zendeskgarden/react-dropdowns';
+import { Avatar, IStatusIndicatorProps, StatusIndicator } from '@zendeskgarden/react-avatars';
+
+export const StatusMenuStory: Story = ({ isCompact }) => {
+  const [selectedType, setSelectedType] = useState<IStatusIndicatorProps['type']>('available');
+
+  return (
+    <Grid>
+      <Row style={{ height: 'calc(100vh - 80px)' }}>
+        <Col textAlign="center" alignSelf="center">
+          <Dropdown selectedItem={selectedType} onSelect={value => setSelectedType(value)}>
+            <Trigger>
+              <Avatar status={selectedType} size={isCompact ? 'small' : 'large'}>
```

Comment:
[nit]

```suggestion
              <Avatar status={selectedType} size={isCompact ? 'small' : 'medium'}>
```

...so that the system stays intact.